### ISSUE TITLE
feature request: change cursor color

### ISSUE THREAD

**Issue by RaananHadar** [MaokaiLin/PowerCursors#5]
first of all, your plugin is extremely useful. I was thinking of writing the very same thing myself. combined with easy motion its truly amazing.

What I would really appreciate if you implemented a minor feature enhancement.
I think it should be possible to change the appearance of the disabled cursors. I believe that a block cursor with different color scheme than the active cursor would improve usability and appeal to the user compared to the current implementation. So I will greatly appreciate if you add the ability to alter this.

thanks.

**Comment by ctf0** [comment]
@RaananHadar +1 to ur idea also is there any chance you can make a new plugin that combins PowerCursors with easy motion ? :+1:

**Comment by babygau** [comment]
+1 to the idea, but I don't think @MaokaiLin will do that, this project has been in active for almost 2 years @ctf0

**Comment by ctf0** [comment]
@babygau yes i know but i was hopping @RaananHadar could release his version as atm the plugin doesnt work with ST 3095 and dont think it will with any future releases.

**Comment by babygau** [comment]
@ctf0, what does not work for you, I'm curious, because I'm using `PowerCursor` with build 3095 without problem

**Comment by ctf0** [comment]
@babygau i dont know why but i cant use the multi cursor anymore , it simply doesn't work

**Comment by babygau** [comment]
@ctf0 make sure you don't have keybinding conflict with other plugins such as `Easy Motion`. I'm using `Power Cursor` and `Ace Jump` without any issues.

**Comment by ctf0** [comment]
will try again then, thanx for the heads up
